### PR TITLE
[WIP] New options page

### PR DESCRIPTION
**Pull request by pradyunsg**
I just started work on the new options page. This is just a quick non-functional mockup of the idea I have in mind. I just wanted to get some opinion on this thing, just to be sure I'm not making something that I'll have to trash later...

While, I realize that I forgot about the translation-related stuff in the current state of the page, it should be trivial to add, given that I reuse the original text, which I obivously intend to.

I just wanted to ask if it is OK if I re-arrange the files used by options.html so that the folder is more organized. (Someone will need to run :eyes: to make sure I don't break something...)

/cc @rafegoldberg @adam-p

### CODE REVIEW

**Comment by pradyunsg** [comment]
Any comments?

**Comment by pradyunsg** [comment]
Knock knock!! Anybody here?

**Comment by adam-p** [comment]
`options-tab-about.html` and `jquery-2.1.4.js` are missing or mislocated in the PR.

**Comment by pradyunsg** [comment]
Yes they are. 

I included the minified version of Jquery but am refering to the un-minified version. Silly me! And options-tab-about never actually got written...

So, Is the design OK? We could put a different markdown theme if needed. Then I'll go about adding all the settings-related JS handling.

File: src/common/test/index.html [hunk 1]
```diff
@@ -36,7 +36,7 @@
   <script src="../options-store.js"></script>
   <script src="../markdown-here.js"></script>
 
-  <script src="../vendor/jquery-1.9.1.js"></script>
+  <script src="../extras/js/jquery-2.1.4.js"></script>
```
Comment (pradyunsg) [inline src/common/test/index.html]: Change to include the minified version...

**Comment by drKnoxy** [comment]
is this dead?

**Comment by pradyunsg** [comment]
Not dead, just stagnant. 

I was waiting for kind of a review from @adam-p or anyone else for that matter, to just give me a picture of what it really needs to be. I would hate to rewrite half the thing all over because it didn't fit some constraint I didn't know.

And, now, I'm having a lot of things on my plate.

I've done some stuff on this, unfinished, unpushed... And as far as I can see, I need to rebase as well :stuck_out_tongue:.

**Comment by pradyunsg** [comment]
@adam-p Ping!!!

I should be able to get some time to work on this later this month. You there?

**Comment by pradyunsg** [comment]
This PR is bitrotted.

**Pull request closed without merging**